mne/beamformer/tests/test_lcmv.py

```diff
@@ -211,6 +213,14 @@ def test_lcmv():
                   data_cov, reg=0.01, pick_ori="max-power", weight_norm=None,
                   max_ori_out='signed')
 
+    # Test if wrong channel selection is detected in application of filter
+    evoked_ch = deepcopy(evoked)
+    evoked_ch.pick_channels(evoked_ch.ch_names[:-1])
+    sfilter = make_lcmv_filter(evoked.info, forward_vol, data_cov, reg=0.01,
```

Comment:
There is a private function ``_apply_lcmv``, could this lead to confusion?

(spatial) filter(s) could be mixed up with data filters - was that your initial concern, @dengemann ?